knowledge/biology/mitochondrion/qna.yaml

```diff
@@ -0,0 +1,24 @@
+created_by: Charles McCleland
+domain: biology
+seed_examples:
+- answer: A mitochondrion is an organelle found in the cells of most eukaryotes, such as animals, plants and fungi.
+  question: What are Mitochondria?
+- answer: They were discovered by Albert von Kölliker in 1857
+  question: Who discovered Mitochondria?
+- answer: Some cells in some multicellular organisms lack mitochondria (for example, mature mammalian red blood cells).
+  question: Do all cells have mitochondria?
+- answer: Mitochondria have a double membrane structure and use aerobic respiration to generate adenosine |
```

Comment:
I believe the `|` at the end of this line is breaking the linter. normally it would be something like:
```
- answer: |
     Mitochondria have a double membrane structure and use aerobic respiration to generate adenosine
     triphosphate (ATP), which is used throughout the cell as a source of chemical energy.
   question: |
     What are the physical characteristics of mitochondria?
```

Reply:
ok, I will fix it.  Documentation could be more clear on this.  I added the | to avoid ending the line with a space.